src/scripts/fetchGames.ts

```diff
@@ -1,6 +1,6 @@
 import { notFound } from 'next/dist/client/components/not-found';
 
-export const fetchGames = async () => {
+const fetchGames = async () => {
   const res = await fetch('http://localhost:3000/api/games', { cache: 'no-store' });
```

Comment:
żeby to pobieranie gier nam działało na vercelu, musimy zmienić ten endpoint w zależności czy jesteśmy na dev czy na prod. Możemy to osiągnąć za pomocą `process.env.NODE_ENV` i w zależności od tej właściwości podstawiać api backend url